e2e/step_definitions/programCatalog.js

```diff
@@ -31,4 +34,23 @@ defineSupportCode(({Given, When, Then, setDefaultTimeout}) => {
             ProgramCatalog.courseCount = count;
         });
     });
+
+    Then(/^the text "(.*)" is typed into the search bar$/, text => {
```

Comment:
Please, modify the regex so that it works if there is a " character in the search term.
([^"].*)